AI_ML/OPENAI/DALLE_IMAGE_GENERATOR/DALLE_IMAGE_GENERATOR.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+from flojoy import flojoy, DataContainer, Image
+from typing import List
+import openai
+from PIL import Image as PilImage
+import requests
+from io import BytesIO
+import os
+
+
+@flojoy
+def DALLE_IMAGE_GENERATOR(
+    prompt: str
+) -> Image:
+    """
+    The DALLE_IMAGE_GENERATOR node takes a prompt and generates an image
+    using OpenAI's DALL-E model.
+    The prompt should be a sentence describing the image you want to generate.
+    The image will be returned as a DataContainer with the type 'image'.
+
+    Parameters
+    ----------
+    prompt: string
+        A sentence describing the image you want to generate.
+    """
+    openai.api_key = os.environ.get("OPENAI_API_KEY")
+    result = openai.Image.create(prompt=prompt, n=1, size="1024x1024")
```

Comment:
these should be optional params, the user should get to specify the prompt, n, and size of the output